server/src/dtos/person.dto.ts

```diff
@@ -181,6 +181,52 @@ export class PeopleResponseDto {
   hasNextPage?: boolean;
 }
 
+export class CreateFaceDto {
+  @ValidateUUID()
+  personId!: string;
+
+  @ValidateUUID()
+  assetId!: string;
+
+  @ApiProperty({ type: 'integer' })
+  @IsNotEmpty()
+  @IsNumber()
+  imageWidth!: number;
+
+  @ApiProperty({ type: 'integer' })
+  @IsNotEmpty()
+  @IsNumber()
+  imageHeight!: number;
```

Comment:
It would be preferable to move this server-side and infer it from the file. Either by getting metadata from sharp or by just storing the preview dimensions in the database in the asset_files table.